backend/conduit/organizations/views.py

```diff
@@ -78,7 +78,7 @@ def delete_organization(slug):
 @blueprint.route('/api/organizations/<slug>/follow', methods=('POST',))
 @jwt_required
 @marshal_with(organization_schema)
-def follow_an_organization(slug):
+def follow_organization(slug):
```

Comment:
Pull code from staging and test your routes again. That way I won't have to merge in duplicated code plss.